docs/help/support-guidelines.md

```diff
@@ -1,5 +1,86 @@
 # Support guidelines
 
+This page explains the support mechanisms we provide.
+
 !!! warning
 
-    Under Construction
+    Before asking any help, search and read this documentation site carefully.
+    Also, understand and follow our [discussion guidelines](../contributing/discussion-guidelines/index.md) during your discussions.
+
+Choose appropriate resources depending on what kind of help you need:
+
+- Documentation sites
+  - Various information
+- GitHub Discussions
+  - Questions
+  - Unconfirmed bugs
+  - Feature requests
+  - Design discussions
+- GitHub Issues
+  - Confirmed bugs
+- Slack
+  - Instant messaging between contributors
+- ROS Discourse
+  - General topics that should be widely announced
+
+The following sections are detailed descriptions of each resource.
```

Comment:
```suggestion
Following sections contain detailed descriptions of each these resources.
```

Since it is the first time we are referring to `Following sections` I dropped the `the` before it.

Reply:
Grammarly raises warnings. :thinking: 

![image](https://user-images.githubusercontent.com/31987104/157448159-087eff73-f2ad-4c3d-80f6-502df71b2470.png)

![image](https://user-images.githubusercontent.com/31987104/157448182-345f75b1-f889-4e01-a167-8185b5cbe29b.png)

Reply:
https://www.grammarly.com/blog/articles-with-plural-nouns/

> Plural nouns can take either a definite article or no article at all. I think the is unnecessary there.

Reply:
Grammatically speaking, it should be `The following`. However, I would argue that the line can be removed entirely, since it is clear from context that the following sections provide detail on the various communication channels available for support.

Reply:
Hmm, I see, thank you.

But I want people not to stop here and proceed to the following sections.
So how about changing here like the following?

```
Confirm the detailed descriptions of each resource in the following sections.
```

or,

```
```
See the following sections for the detailed descriptions of each resource.
```

Reply:
Hmm. I see your point, but at the same time I think most people would naturally continue reading down the page. 

If we added links to the detailed description section for each communication channel in the list, would that be a clear enough indication that there is more detail below? 🤔

Reply:
@LalithVipulananthan How about this? 5fcad549196a5cb9a7c1392b84746a7099545b86

Reply:
Excellent! 👍🏾

Reply:
Actually, I tried this first but didn't look good because there was little content. :sweat_smile: 
But now it looks good!

Reply:
@xmfcx The bold styles were removed because links are added instead. Is it okay for you?
If so, please resolve the conversation! :pray: 

![image](https://user-images.githubusercontent.com/31987104/157783213-8bc94431-565a-4714-96c7-e8fd2fcbf802.png)